docs/deepsemgrep/deepsemgrep-examples.md

```diff
@@ -7,77 +7,92 @@ toc_max_heading_level: 5
 
 ```sh
 semgrep --config pro.yaml . --pro
 ```
+:::
 
 ### JavaScript and TypeScript
 
-Here, Semgrep matches `dangerous(“Select * from “ + user_input)`, because `user_input` is obtained by calling `get_user_input`. However, it does not match the similar call using `still_user_input`, because its analysis does not cross function boundaries to know that `still_user_input` is a wrapper function for `user_input`.
+Here, Semgrep OSS matches `dangerous(“Select * from “ + user_input)`, because `user_input` is obtained by calling `get_user_input`. However, Semgrep OSS does not match the similar call using `still_user_input`, because its analysis does not cross function boundaries to know that `still_user_input` is a wrapper function for `user_input`.
 
 <iframe title="Semgrep example no prints" src="https://semgrep.dev/embed/editor?snippet=Po9p" width="100%" height="432" frameborder="0"></iframe>
 
-Semgrep Pro matches both dangerous calls, because it does cross function boundaries. In fact, with Semgrep Pro, the taint rule can track calls to `get_user_input` over multiple jumps in multiple files.
+Semgrep Pro matches both dangerous calls because it does cross function boundaries. In fact, with Semgrep Pro, the taint rule can track calls to `get_user_input` over multiple jumps in multiple files.
 
-Try toggling the [Semgrep Pro Engine](https://semgrep.dev/s/Po9p) to see it match in the playground. To run Semgrep Pro in the cloned [Semgrep Pro Engine testing repository](https://github.com/returntocorp/semgrep-pro-tests). Go to `docs/taint_tracking/javascript` and run the following command:
+:::tip Try it out
+Enable the **Semgrep Pro Engine beta** <i class="fa-solid fa-toggle-large-on"></i> toggle in the following link to an [example of dangerous taint](https://semgrep.dev/s/Po9p). To run Semgrep Pro Engine in the cloned [Semgrep Pro Engine testing repository](https://github.com/returntocorp/semgrep-pro-tests). Go to `docs/taint_tracking/javascript` and run the following command:
 
 ```sh
 semgrep --config pro.yaml . --pro
 ```
+:::
 
 #### ES6 and CommonJS
 
-The JavaScript and TypeScript ecosystems contain various ways for importing and exporting code, Semgrep Pro is able to track dataflow through ES6 imports/exports and some CommonJS export paths (See known limitations below). 
+The JavaScript and TypeScript ecosystems contain various ways for importing and exporting code, Semgrep Pro Engine can track dataflow through ES6 imports or exports and some CommonJS export paths (See known limitations below).
 
-<b>ES6</b>
+##### ES6
 
-Semgrep is able to track data through the definition of exports for es6:
+TODO Semgrep is able to track data through the definition of exports for es6:
 
 ```js
 export function readUser() {
     return get_user_input("example")
 }
 ```
 
-Semgrep is able to follow the dataflow when it is imported in another location:
+Semgrep is able to follow the dataflow when it is imported into another location:
```

Comment:
@milanwilliams please, instead of just Semgrep, is this Semgrep OSS or Semgrep Pro Engine? (We need to be specific.)

Reply:
```suggestion
Semgrep Pro Engine is able to follow the dataflow when it is imported into another location:
```